test/extensions/filters/network/thrift_proxy/integration_test.cc

```diff
@@ -148,6 +149,11 @@ class ThriftConnManagerIntegrationTest
   // while oneway's are handled by the "poke" method. All other requests
   // are handled by "execute".
   FakeUpstream* getExpectedUpstream(bool oneway) {
+    int upstreamIdx = getExpectedUpstreamIdx(oneway);
```

Comment:
i don't think clang tidy will allow this local var name.

Reply:
`int idx =`

Reply:
ah never mind it's a thing below

Reply:
didnt change as its used in the existing code